bluebrain/repo-bluebrain/packages/neurodamus-model/package.py

```diff
@@ -54,7 +59,7 @@ class NeurodamusModel(SimModel):
     resource(
```

Comment:
This should still be a full-blown dependency if it changes over time!

Reply:
I think that in the near future this will be deleted and all the mod files should come from the same repo, so we won't have this mess